File: fine_tuning/results/test_output_files/question-24-pass-1.py
def countRepeatingDigits(N):
    count = 0
    for i in range(1, 10):
        if (N // i) % 10 == 0:
            count += 1
    print(count)


if __name__ == "__main__":
    N = 123456
    countRepeatingDigits(N)

File: results/feedback_pipeline/cpp_results/cpp_files_after_feedback/question-24-pass-1.cpp
#include <bits/stdc++.h>
using namespace std;
int countRepeatingDigits(int N) {
    int count = 0;
    int digit = N % 10;
    while (N > 0) {
        int next_digit = N % 10;
        if (next_digit == digit)
            count++;
        N /= 10;
    }
    return count;
}
int main() {
    int N;
    cin >> N;
    cout << countRepeatingDigits(N) << endl;
    return 0;
}